ssf/frame.proto

```diff
@@ -0,0 +1,30 @@
+// As we're using required fields, this is proto2 syntax.
+syntax = "proto2";
+
+package ssf;
+
+// A fixed-length frame message for SSF span messages in a
+// non-delimited serial format (streaming network sockets, on-disk
+// representation), this is meant to be sent right before an
+// SSFSpanCollection message, and specifies the length of that message
+// in bytes.
+//
+// A protobuf-serialized SSFFrameHeader message is always expected to
+// be 10 bytes long.  As the frame message is defined to be a fixed
+// width, it's not expected to change.
+//
+// A typical stream with framed messages looks like:
+//   [SSFFrameHeader, 10 bytes][SSFSpanCollection, n bytes][SSFFrameHeader, 10 bytes][...]
```

Comment:
10? I assume some of that is protobuf encoding over the `4` bytes of each `fixed32`?

Reply:
Determined experimentally: `(echo 'version:0'; echo 'length:1109283192') | protoc --encode=ssf.SSFFrame ssf/frame.proto | wc -c` 😭 
 
I'll try to do this more rigorously (: